### PR TITLE
SCA23

### PR DESCRIPTION
**Pull request by tccaws**
*Issue #, if available:*
**SCAsia 2023 workshop update to Lab III.**

*Description of changes:*

- Replaced SC22 Lab III container building automation with Lab III container building automation content  from ISC22.
- Changed references to ISC22 to SCA23.
- Added ECR repository creation command to step 1.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/05-cicd-pipeline/04-codebuild.md [hunk 1]
```diff
@@ -11,9 +11,7 @@ AWS CodeBuild is a fully managed continuous integration service that compiles so
 
 With CodeBuild, you don’t need to provision, manage, and scale your own build servers
 
-
-
-1. Open the [AWS CodeBuild console](https://console.aws.amazon.com/codebuild/home). Double check that you are using CodeBuild in the same AWS Region that you have used in the previous steps.
+1. In the AWS Management Console search bar, type and select **CodeBuild**. Double check that you are using CodeBuild in the same AWS Region that you have used in the previous steps.
```
Comment (mhuguesaws) [inline content/05-cicd-pipeline/04-codebuild.md:14]: No. Keep the old language and direct link to codebuild.

File: content/05-cicd-pipeline/05-codepipeline.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ In the first section of this lab you created a CodeCommit repo and created a sam
 
 In this section, you will create a pipeline with a source and build stage to automate the container build and push to Amazon ECR whenever there are any changes in the source CodeCommit repository.
 
-1. Open the [AWS CodePipeline console](https://console.aws.amazon.com/codepipeline/home).
+1. In the AWS Management Console search bar, type and select **CodePipeline**.
```
Comment (mhuguesaws) [inline content/05-cicd-pipeline/05-codepipeline.md:16]: Keep the old language.

**Review by mhuguesaws — COMMENTED** [review]
See comments.

File: content/05-cicd-pipeline/04-codebuild.md (hunk 1 above)
Comment (tccaws) [reply content/05-cicd-pipeline/04-codebuild.md:14]: That is a difference between the ICS22 content, and the SC22 content. Which got reverted when I overwrote the SC22 file with the file from ICS22.

File: content/05-cicd-pipeline/05-codepipeline.md (hunk 2 above)
Comment (tccaws) [reply content/05-cicd-pipeline/05-codepipeline.md:16]: That is another difference between the ICS22 content, and the SC22 content. Which got reverted when I overwrote the SC22 file with the file from ICS22.

**Comment by tccaws** [comment]
I'll make those changes and resubmit PR.

**Comment by tccaws** [comment]
Made the changes as suggested.

**Pull request merged**